book.js

```diff
@@ -15,7 +19,19 @@ for (var i = 0; i < maxQuantity; i++) {
 }
 
 $quantity.onchange = function() {
-  $total.value = unitPrice * $quantity.value;
+  $subtotal.value = unitPrice * $quantity.value;
+  $total.value = Number($subtotal.value) + Number($shipping.value);
 };
 
 $quantity.onchange(null);
+
+$shippingCheckbox.onclick = function() {
+  if ($shippingCheckbox.checked) {
+    $shipping.value = intShippingPrice;
+  } else {
+    $shipping.value = 0;
```

Comment:
You are defaulting to 0, but maybe the default shipping can change in the future. I think you should read this value from a `data-` attribute like you're doing with the `intShippingPrice`. Perhaps you can add on the same element, a `data-base-shipping` data attribute.